ivi-layermanagement-examples/multi-touch-viewer/src/multi-touch-viewer.c

```diff
@@ -646,35 +646,29 @@ usage(int status)
 {
     printf("usage: multi-touch-viewer [OPTION]\n");
     printf("  -p : print received touch point\n");
+    printf("  -s : select the type of surface, default is the ivi_app_surface.\n");
+    printf("       -s=wl_shell: to using the wl_shell_surface\n");
+    printf("       -s=ivi_app: to using the ivi_app_surface\n");
+    printf("  -h : show this help and exit\n");
     exit(status);
 }
 
 int
 main(int argc, char **argv)
 {
-    _UNUSED_(argc);
-    _UNUSED_(argv);
-    struct touch_event_test_params params;
+    if(isOptionAvailable("-h", argc, argv)){
+        usage(EXIT_SUCCESS);
+    }
 
+    struct touch_event_test_params params;
     memset(&params, 0x00, sizeof params);
 
```

Comment:
I recommend to have the option parsing code separated. Right now option parsing is propagated to different functions. 
Also please getopt for option parsing. Please see this example below.

https://github.com/COVESA/wayland-ivi-extension/blob/f6911a11dc911a5bcb380d0895db6cfd533a3569/ivi-layermanagement-examples/layer-add-surfaces/src/layer-add-surfaces.c#L190